infrastructure/static-site.tf

```diff
@@ -151,6 +169,17 @@ resource "aws_route53_record" "tokyorust" {
   }
 }
 
+resource "aws_route53_record" "root_tokyorust" {
+  zone_id = aws_route53_zone.tokyorust.id
+  name    = local.host_name
+  type    = "A"
+  alias {
+    name = aws_cloudfront_distribution.s3_distribution.domain_name
+    zone_id = aws_cloudfront_distribution.s3_distribution.hosted_zone_id
+    evaluate_target_health = false
+  }
+}
+
 resource "aws_iam_policy" "tokyorust-static-deployer" {
```

Comment:
Actually, could you delete this instead of modify it? A while back I decided to manage the deployment policy elsewhere in order to make privilege escallation attacks harder. I forgot to go back and delete this configuration.

Reply:
Done - I've deleted the resources:
resource "aws_iam_policy" "tokyorust-static-deployer"
resource "aws_iam_user_policy_attachment" "tokyorust-static-deployer"